cleanup-passes/travis.ts

```diff
@@ -53,18 +59,30 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
 
   const tools = [
     'bower',
-    'polylint',
-    'web-component-tester'
+    'polymer-cli',
+    'web-component-tester',
   ];
 
-  const beforeScript = [
+  // Override install script for all elements
+  const install = [
     `npm install -g ${tools.join(" ")}`,
-    'bower install',
-    'polylint'
+    'polymer install --variants',
   ];
+  if (Array.isArray(travis.install)) {
+    const installEqual = travis.install.reduce(
+      (acc, s, idx) => { return acc && (install[idx] === s) }, true);
```

Comment:
Was confusing to me as well at first. I was thinking it might be useful in the `util.ts` file but I can just put it here for now.